src/Compute/Compute/ChangeLog.md

```diff
@@ -23,6 +23,7 @@
 * Added `-Hibernate` switch parameter to `Stop-AzVmss` default parameter set. 
 * For `Get-AzVmRunCommand`, a bug is fixed to work when returning a list of RunCommands [#22403]
 * Updated Azure.Core to 1.34.0.
+* Fix for `Update-AzVMSS` to update ImageReferenceSKU [#22195]
```

Comment:
Update "Fix" to "Fixed".

Reply:
Done

Reply:
Hi @haagha , please double-check the changes. You didn't merge it correctly.